.github/workflows/report.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           poetry run coverage combine --keep coverage-python3.9*/.coverage
           cp .coverage ../ || true
           cp lint-python3.9/.lint.txt ../ || true
+          cp lint-python3.9/.lint.json ../ || true
```

Comment:
maybe also add a comment above why we ignore the result (because we validate it in the next step)